src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,84 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter SIMPLE_DATE_FORMAT = DateTimeFormatter
+            .ofPattern("dd.MM.yyyy");
+    private static final int ZERO_INDEX = 0;
+    private static final int FIRST_INDEX = 1;
+    private static final int SECOND_INDEX = 2;
+    private static final int THIRD_INDEX = 3;
```

Comment:
bad naming, name after roles, not values. Value can change later. change to NAME_INDEX, AMOUNT_INDEX etc